manufacturing-ai-suite/wind-turbine-anomaly-detection/docs/user-guide/Overview.md

```diff
@@ -1,87 +1,8 @@
-# Wind Turbine Anomaly Detection
+# Overview

```

Comment:
With this you wont be seeing this name showing up in the highlighted section in below snap when published?

![Uploading image.png…]()